projects/v3/src/app/pages/assessment-mobile/assessment-mobile.page.ts

```diff
@@ -85,12 +88,16 @@ export class AssessmentMobilePage implements OnInit {
   }
 
   async saveAssessment(event) {
+    this.savingText = 'Saving...';
     await this.assessmentService.saveAnswers(event.assessment, event.answers, event.action, this.assessment.pulseCheck).toPromise();
     if (!event.assessment.inProgress) {
       this.notificationsService.assessmentSubmittedToast();
       // get the latest activity tasks and refresh the assessment submission data
       this.activityService.getActivity(this.activityId);
+      this.savingText = 'Last saved ' + this.utils.getFormatedCurrentTime();
       return this.assessmentService.getAssessment(this.assessment.id, this.action, this.activityId, this.contextId, this.submissionId);
+    } else {
+      this.savingText = 'Last saved ' + this.utils.getFormatedCurrentTime();
```

Comment:
the savingText is changed for both `if` and `else`, why don't we just bring it out of the `if,else` then?